llvm/test/CodeGen/SPIRV/hlsl-intrinsics/splitdouble_vector.ll

```diff
@@ -0,0 +1,14 @@
+; RUN: opt -S -scalarizer -mtriple=spirv-vulkan-library %s 2>&1 | llc -verify-machineinstrs -O0 -mtriple=spirv-unknown-unknown -o - | FileCheck %s
```

Comment:
I don't think this file should exist. I don't understand why we are using `opt` the `-scalarizer` pass and why we aren't running the output into the SPIRV validator via `spirv-val`